### PR TITLE
Update cucumber-reporting to 1.4.0

### PR DESCRIPTION
**Pull request by tommywo**

### CODE REVIEW

File: src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java [hunk 1]
```diff
@@ -115,12 +115,18 @@ public void execute() throws MojoExecutionException {
         }
 
         try {
-            ReportBuilder reportBuilder = new ReportBuilder(list, outputDirectory, "", buildNumber, projectName, skippedFails, pendingFails, undefinedFails, missingFails, enableFlashCharts, false, false, "", false, false);
+            Configuration configuration = new Configuration(outputDirectory,projectName);
+            configuration.setBuildNumber(buildNumber);
+            configuration.setStatusFlags(skippedFails,pendingFails,undefinedFails,missingFails);
+            configuration.setParallelTesting(parallelTesting);
+            configuration.setRunWithJenkins(false);
```
Comment (damianszczepanik) [inline src/main/java/net/masterthought/cucumber/CucumberReportGeneratorMojo.java]: probably not needed, this is default value

**Comment by damianszczepanik** [comment]
Sorry about delay, I was not notified about this PR. Can you merge commits into one

**Comment by tommywo** [comment]
i squashed commits into one and bumped cucumber-reporting version to 1.4.0

**Comment by damianszczepanik** [comment]
Sorry for the delay

**Pull request merged**